pkg/scheduler/cache/cache.go

```diff
@@ -851,4 +858,17 @@ func (sc *SchedulerCache) recordPodGroupEvent(podGroup *schedulingapi.PodGroup,
 		return
 	}
 	sc.Recorder.Eventf(pg, eventType, reason, msg)
+
+	jobLister := sc.jobInformer.Lister()
+	batchJob, err := jobLister.Jobs(podGroup.Namespace).Get(podGroup.Name)
+	if err != nil {
+		klog.Errorf("Failed to get VolcanoJob  <%s/%s>: %v",
+			podGroup.Namespace, podGroup.Name, err)
+	}
+	if batchJob == nil {
+		klog.Infof("VolcanoJob <%s/%s> not found",
+			podGroup.Namespace, podGroup.Name)
+	} else {
+		sc.Recorder.Eventf(batchJob, eventType, reason, msg)
+	}
```

Comment:
Adding job event logic into `recordPodGroupEvent` funciton seems a little strange. Can you describe the scenario or consideration?

Reply:
> Adding job event logic into recordPodGroupEvent funciton seems a little strange

Volcano scheduler should not aware any kind of k8s, it only does scheduling based on PodGroup.

Reply:
When podgroup can't schedule due to resource limit ,we also need to sync event info to volcano job which help users troubleshoot problems. What do you think?

Reply:
> When podgroup can't schedule due to resource limit ,we also need to sync event info to volcano job which help users troubleshoot problems. What do you think?

Sound reasonable in some degree. It' a feaure in v1.2 which has been proposed by @merryzhou at https://github.com/volcano-sh/volcano/issues/1094 @k82cn

Reply:
we should not bind podgroup to job; only pod & podgroup should be used in scheduler.